src/nvim/mouse.c

```diff
@@ -245,6 +246,16 @@ static int get_fpos_of_mouse(pos_T *mpos)
   return IN_BUFFER;
 }
 
+/// Execute mouse move autocmd.
+static void do_move_autocmd(int screenrow, int screencol) {
+	Dictionary data = ARRAY_DICT_INIT;
+	PUT(data, "screenrow", INTEGER_OBJ(screenrow + 1));
+	PUT(data, "screencol", INTEGER_OBJ(screencol + 1));
+	Object obj = DICTIONARY_OBJ(data);
+	apply_autocmds_group(EVENT_MOUSEMOVED, NULL, NULL, false, AUGROUP_ALL, NULL, NULL, &obj);
+	api_free_dictionary(data);
```

Comment:
Not sure if I should be calling `api_free_dictionary(data)` or `api_free_object(obj)` here

Reply:
`api_free_object` , else you need to do `api_free_dictionary(data.dictionary)`